daemon/graphdriver/btrfs/version.go

```diff
@@ -13,9 +13,21 @@ int my_btrfs_lib_version() {
   return -1;
 #endif
 }
+
+char* my_btrfs_build_version() {
+#ifdef BTRFS_BUILD_VERSION
+  return BTRFS_BUILD_VERSION;
+#else
+  return "-";
+#endif
```

Comment:
If we did the same thing with `BTRFS_LIB_VERSION`, we could probably get rid of `btrfs_noversion` completely, right?